README.md

```diff
@@ -53,53 +46,68 @@ See the [documentation](https://help.github.com/en/articles/metadata-syntax-for-
 Most toolkit and CI/CD operations involve async operations so the action is run in an async function.
 
 ```javascript
-import * as core from '@actions/core';
+const core = require('@actions/core');
 ...
-
 async function run() {
-  try { 
+  try {
       ...
-  } 
+  }
   catch (error) {
     core.setFailed(error.message);
   }
 }
-
 run()
 ```
 
 See the [toolkit documentation](https://github.com/actions/toolkit/blob/master/README.md#packages) for the various packages.
 
-## Publish to a distribution branch
+## Package for distribution
+
+GitHub Actions will run the entry point from the action.yml. Packaging assembles the code into one file that can be checked in to Git, enabling fast and reliable execution and preventing the need to check in node_modules.
+
+Actions are run from GitHub repos. Packaging the action will create a packaged action in the dist folder.
+
+Run prepare
+
+```bash
+npm run prepare
+```
 
-Actions are run from GitHub repos so we will checkin the packed dist folder. 
+Since the packaged index.js is run from the dist folder.
 
-Then run [ncc](https://github.com/zeit/ncc) and push the results:
 ```bash
-$ npm run package
-$ git add dist
-$ git commit -a -m "prod dependencies"
-$ git push origin releases/v1
+git add dist
+```
+
+## Create a release branch
+
+Users shouldn't consume the action from master since that would be latest code and actions can break compatibility between major versions.
+
+Checkin to the v1 release branch
+
+```bash
+git checkout -b v1
+git commit -a -m "v1 release"
+```
+
+```bash
+git push origin v1
 ```
 
 Note: We recommend using the `--license` option for ncc, which will create a license file for all of the production node modules used in your project.
 
-Your action is now published! :rocket: 
+Your action is now published! :rocket:
 
 See the [versioning documentation](https://github.com/actions/toolkit/blob/master/docs/action-versioning.md)
 
-## Validate
+## Usage
 
-You can now validate the action by referencing `./` in a workflow in your repo (see [test.yml](.github/workflows/test.yml))
+You can now consume the action by referencing the v1 branch
 
 ```yaml
-uses: ./
+uses: actions/javascript-action@v1
```

Comment:
see prior comment, shouldn't this point to the repository?

Reply:
have added the use case of this action.